src/openzeppelin/tests/mocks/reentrancy_mock.cairo

```diff
@@ -1,93 +1,92 @@
 use starknet::ContractAddress;
 
-#[abi]
-trait IReentrancyGuarded {
-    fn count_external_recursive(n: felt252);
+#[starknet::interface]
+trait IReentrancyGuarded<TContractState> {
+    fn count_external_recursive(ref self: TContractState, n: felt252);
 }
 
-#[abi]
-trait IReentrancyMock {
-    #[view]
-    fn current_count() -> felt252;
-    #[external]
-    fn callback();
-    #[external]
-    fn count_local_recursive(n: felt252);
-    #[external]
-    fn count_external_recursive(n: felt252);
-    #[external]
-    fn count_and_call(attacker: ContractAddress);
-    #[external]
-    fn count();
+#[starknet::interface]
+trait IReentrancyMock<TContractState> {
+    fn current_count(self: @TContractState) -> felt252;
+    fn callback(ref self: TContractState);
+    fn count_local_recursive(ref self: TContractState, n: felt252);
+    fn count_external_recursive(ref self: TContractState, n: felt252);
+    fn count_and_call(ref self: TContractState, attacker: ContractAddress);
 }
 
-#[contract]
+#[starknet::contract]
 mod ReentrancyMock {
-    // OZ modules
     use openzeppelin::security::reentrancyguard::ReentrancyGuard;
-
-    // Dispatchers
-    use super::IReentrancyGuardedDispatcher;
-    use super::IReentrancyGuardedDispatcherTrait;
     use openzeppelin::tests::mocks::reentrancy_attacker_mock::IAttackerDispatcher;
     use openzeppelin::tests::mocks::reentrancy_attacker_mock::IAttackerDispatcherTrait;
-
-    // Other
-    use option::OptionTrait;
     use starknet::ContractAddress;
```

Comment:
We've been putting oz imports below starknet imports, I think we should be consistent with the order.